src/commons/navigationBar/subcomponents/NavigationBarLangSelectButton.tsx

```diff
@@ -35,29 +35,16 @@ const NavigationBarLangSelectButton = () => {
     setIsOpen(false);
   };
 
+  console.log('lang', lang); // FIXME: Not updating
+
```

Comment:
For some reason this change introduces a regression: The currently selected option is cleared on refresh (and defaults back to JS, even if the playground is already set to the correct language).